database/src/main/runs/get_segmentation_measurements.sql

```diff
@@ -0,0 +1,108 @@
+--      If the given segmentation does not exist, exactly one record is returned with the error status.
+--
+-- Parameters:
+--      i_segmentation                                      - segmentation for which the data are bound to
+--
+-- Returns:
+--      status                                              - Status code
+--      status_text                                         - Status text
+--      id_measurement                                      - Measurement id
+--      id_checkpoint_measure_definition                    - Checkpoint measure definition id
+--      id_checkpoint                                       - Checkpoint id
+--      value                                               - Measurement value
+--      checkpoint_name                                     - Checkpoint name
+--      workflow_name                                       - Workflow name
+--      checkpoint_process_start_time                       - Checkpoint process start time
+--      checkpoint_process_end_time                         - Checkpoint process end time
+--      checkpoint_created_by                               - Checkpoint created by
+--      checkpoint_created_at                               - Checkpoint created at
+--      measure_type                                        - Measure type
+--      measure_fields                                      - Measure fields
+--      checkpoint_measure_definition_created_by            - Checkpoint measure definition created by
+--      checkpoint_measure_definition_created_at            - Checkpoint measure definition created at
+--
+-- Status codes:
+--      10                                                  - OK
+--      41                                                  - Segmentation not found
+--
+-------------------------------------------------------------------------------
+DECLARE
+    _key_segmentation   BIGINT;
+BEGIN
+    _key_segmentation = runs._get_key_segmentation(i_segmentation);
+
+    IF _key_segmentation IS NULL THEN
+        status := 41;
+        status_text := 'Segmentation not found';
+        RETURN NEXT;
+        RETURN;
+    END IF;
+
+    RETURN QUERY
+    SELECT
+      10,
+      'OK',
+      m.id_measurement,
+      m.key_checkpoint_measure_definition AS id_checkpoint_measure_definition,
+      m.key_checkpoint AS id_checkpoint,
+      m.value,
+      ch.checkpoint_name,
+      ch.workflow_name,
+      ch.process_start_time AS checkpoint_process_start_time,
+      ch.process_end_time AS checkpoint_process_end_time,
+      ch.created_by AS checkpoint_created_by,
+      ch.created_at AS checkpoint_created_at,
+      chmd.measure_type,
+      chmd.measure_fields,
+      chmd.created_by AS checkpoint_measure_definition_created_by,
+      chmd.created_at AS checkpoint_measure_definition_created_at
+    FROM runs.checkpoints ch
```

Comment:
Just a remark - funny, the cultural influence, that _ch_ is perceived as one letter for Czechs 😄

Reply:
It was not meant as a impulse for change. It was just a funny (or geeky) observation. 😄